readthedocs/config/models.py

```diff
@@ -0,0 +1,37 @@
+"""Models for the response of the configuration object."""
```

Comment:
We can refactor these to be dataclasses when we are fully running py3 :sunglasses:

Reply:
Done https://github.com/rtfd/readthedocs.org/issues/4525